dbms/src/Server/Client.cpp

```
@@ -336,6 +337,7 @@ class Client : public Poco::Util::Application
 
         if (is_interactive)
         {
+            query_id = "";
```

I think it's better to throw an exception instead of silent ignore.